packages/component-library/stories/CivicStoryCard.story.js

```diff
@@ -53,6 +66,27 @@ const tdvDemo = () => (
     </CivicStoryCard>
   </Container>
 );
+const collapsableDemo = () => (
+  <Container>
+    <CivicStoryCard title={'Dogs x Income'}>
+      <Collapsable>
+        <Collapsable.Section>
+          <div style={{ display: 'flex', justifyContent: 'space-around' }}>
+            <HorizontalBarChart
+              data={data}
+              dataKey={dataKey}
+              dataValue={dataValue}
+              dataKeyLabel={dataKeyLabel}
+            />
+          </div>
+        </Collapsable.Section>
+        <Collapsable.Section hidden>
+          <p className="Description">{wallOfRichText}</p>
+        </Collapsable.Section>
```

Comment:
Even though `Collapsable.Section` is a glorified div, I like the semantics here. It makes it really clear to read.